server.go

```diff
@@ -1,15 +1,110 @@
 package main
 
 import (
-    "net/http"
-    "github.com/labstack/echo"
-    "github.com/labstack/echo/engine/standard"
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/sqlite"
+	"github.com/labstack/echo"
+	"net/http"
+	"time"
 )
 
+type Users struct {
+	gorm.Model
+	User_Id      int    `gorm:"AUTO_INCREMENT"`
+	Name         string `gorm:"primary_key"`
+	Batch        string
+	is_master    string
+	EnrollmentNo string
+	Bio          string
+	CreatedAt    time.Time
+	UpdatedAt    time.Time
+}
+
+type Hubs struct {
+	gorm.Model
+	Hub_Id    int    `gorm:"AUTO_INCREMENT"`
+	Name      string `gorm:"primary_key"`
+	Bio       string
+	CreatedAt time.Time
+	UpdatedAt time.Time
+}
+
+type Events struct {
+	gorm.Model
+	Event_Id    int    `gorm:"AUTO_INCREMENT"`
+	Name        string `gorm:"primary_key"`
+	Event       string
+	Description string
+	Date        time.Time
+	CreatedAt   time.Time
+	UpdatedAt   time.Time
+}
+
+type Announcements struct {
+	gorm.Model
+	hub_id      int
+	description string
+	CreatedAt   time.Time
+	UpdatedAt   time.Time
+}
+
+type Roles struct {
+	gorm.Model
+	user_id   int
+	hub_id    int
+	Role      string
+	CreatedAt time.Time
+	UpdatedAt time.Time
+}
+
```

Comment:
Instead of rewriting the models everytime, create a `models.go` file and import them where ever required.